src/services/assetService.ts

```diff
@@ -107,6 +107,9 @@ export class AssetService {
         return assets.map((asset) => {
             if (!asset.path.toLowerCase().startsWith("http://") && !asset.path.toLowerCase().startsWith("https://")) {
                 asset.path = "file:" + asset.path;
+                if (encodeURI(asset.name) !== asset.name) {
```

Comment:
I would update this to always just return the path as the encodedURI vs. checking.